pages/index.js

```diff
@@ -147,6 +179,20 @@ export default function Home({
               name="income"
               ariaLabel="income-select"
               dataCy="income-select-picker"
+              defaultValue={situation.income}
+              onChange={() => {
+                fetch(`${process.env.NEXT_PUBLIC_BASE_URL}/api/situation`, {
+                  method: "post",
+                  headers: {
+                    "Content-Type": "application/json",
+                  },
+                  body: JSON.stringify({
+                    situation: {
+                      income: document.getElementById("income-select").value,
+                    },
+                  }),
+                });
+              }}
```

Comment:
It would be easier to manage these values as state. Then on state change we can post the entire situation object to /api/situation. Then we don't need to write a post every time we add a situation variable and we can read situation from state

Reply:
lines 98 - 195 in index.js. Not sure why it didn't take

Reply:
You are right to move all of this to the state, it does make it easier and there is far less repetition now.